bin/checkstyle.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -x
```

Comment:
wtf is this?

Reply:
not sure. i always copy and pasta

Reply:
## 2.3.2. Debugging on part(s) of the script
```
--
set -x			# activate debugging from here
w
set +x			# stop debugging from here
--
```

Found some good stuff here : http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_02_03.html

Reply:
cool thanks I get it now